ops/storage-ddls/cassandra-setup.cql

```diff
@@ -140,6 +140,27 @@ CREATE INDEX ON computedtopics (period);
 CREATE INDEX ON computedtopics (computedfeatures);
 CREATE INDEX ON computedtopics (periodtype);
 
+CREATE TABLE topplaces (
+    placeid text,
+    placename text,
+    periodstartdate timestamp,
+    periodenddate timestamp,
+    periodtype text,
+    period text,
+    mentions int,
+    PRIMARY KEY ((placeid), periodstartdate, periodenddate)
```

Comment:
Pardon the remedial question, but does this extra set of parenthesis around `placeid` mean something in Cassandra?